drizzlepac/haputils/product.py

```diff
@@ -529,6 +538,81 @@ def copy_exposure(self, filename):
         return edp_filename
 
 
+class GrismExposureProduct(HAPProduct):
+    """ A Grism Exposure Product is an individual Grism/Prism exposure/image (flt/flc).
+
+        The "grism_edp" is short hand for GrismExposureProduct.
+    """
+    def __init__(self, prop_id, obset_id, instrument, detector, filename, filters, filetype, log_level):
+        super().__init__(prop_id, obset_id, instrument, detector, filename, filetype, log_level)
+
+        self.info = '_'.join([prop_id, obset_id, instrument, detector, filename, filters, filetype])
+        self.filters = filters
+        self.full_filename = self.copy_exposure(filename)
+
+        # Open the input FITS file to mine some header information.
+        # and make sure the WCS is up-to-date
+        hdu_list = fits.open(filename)
+        self.mjdutc = hdu_list[0].header['EXPSTART']
+        self.exptime = hdu_list[0].header['EXPTIME']
+
+        drizcorr = hdu_list[0].header['DRIZCORR']
```

Comment:
See previous notes on my questions about the use of this logic in hapsequencer.

Reply:
Same as above
Changed the if/else to just an if: if drizcorr is equal to OMIT, then updatewcs with use_db=True.